ladder/teams/README.md

```diff
@@ -0,0 +1,67 @@
+individuals can be added to the team's mentor list in the team configuration file here. Mentors are no longer part of the auto-assigned PR review rotation, 
+but they can still keep team approval access. Active Mentors of a team are expected to provide guidance to others in the team and seek reviews they 
+should participate in for the code owner area.
+
+## Don't Forget to Set Your GitHub Status
+When active team members are not available for an extended period of time (for example they are on vacation for several days), 
+they are encouraged to set themselves as busy in their [GitHub status](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status). If your status is busy GitHub will skip you to the pull request review rotation for the teams you are on.
+
+Note: Not everyone should be busy all at the same time on a GitHub team if it can be avoided. Cilium Team Management automation puts 
+some guardrails on this and lets the Cilium project know if there aren't sufficient number of reviewers. The GitHub organization owners (or someone 
+delegated to keep tabs on this) will periodically check to see if teams are meeting the expectations of number of people 
+available for review assignment and will reach out to teams if there appears to be a chronic shortfall.
+
+## Team Configuration File Format
+
+Each Cilium GitHub team that wants to self-service their team members (and mentors!) can have a file in this directory named `TEAM_NAME.yaml`
+The file contents as the form of
+```
+# Team members with rights such as commit and approval, as appropriate for the team.
+# They participate in automatic Pull Requests reviews rotation,
+# unless they also are listed as mentors below in this file.
+members:
+ - username_01
+ - username_02
+ - username_03
+
+# Contributors with experience in the team area.
+# They don't have team rights such as commit and approval,
+# unless they are also listed as members above.
+# They are never part of the automatic PRs reviews rotation.
+mentors:
+ - username_02
+ - username_04
+```
+
+In this mockup team file, `username_02` is both a member and a mentor and would be excluded from auto-assignment of PR reviews, but would 
+have other team rights such as commit and approval for changes to files owned by this team.
+
+`username_01` and `username_03` are active team members participating in the automatic PR review rotation.
+
+`username_04` is listed as a mentor, but not a member. This is here as recognition that historically the person participated in the team 
+and is willing to be contacted for historical experience when needed, but doesn't have normal team member access to approve.
+Listing non-active members as mentors is a way to keep institutional knowledge as to who historically provided insight about code functionality to newer 
+members on that team.
```

Comment:
I think this is trying to re-explain the mentor role again and we should probably just keep the info in this section restricted to the formatting rather than the explanation. The previous sections already described what a mentor is, so if that original text is insufficient then let's fix it. Otherwise let's just drop the explanation here. 
```suggestion
`username_04` is listed as a mentor, but not a member. This person doesn't have normal team member access to approve. However, this can be useful to document who may have historical insight into code functionality owned by this team.
```